pydoctor/epydoc2stan.py

```diff
@@ -139,11 +139,11 @@ def resolve_identifier_xref(self, identifier: str, lineno: int) -> str:
 
         if identifier == fullID:
             self.obj.report(
-                "invalid ref to '%s' not resolved" % (identifier,),
+                "invalid ref to \"%s\" not resolved" % (identifier,),
                 'resolve_identifier_xref', lineno)
         else:
             self.obj.report(
-                "invalid ref to '%s' resolved as '%s'" % (identifier, fullID),
+                "invalid ref to \"%s\" resolved as '%s'" % (identifier, fullID),
```

Comment:
There are two quoted values in this string, but only the first one was updated.

Note that #128 is about this message being unclear, so it would be a good idea to either improve the phrasing now as well or to move these message updates into a new PR.

Reply:
I will revert this change. It was a stupid change. Thanks!